keps/sig-network/2593-multiple-cluster-cidrs/README.md

```diff
@@ -791,6 +791,12 @@ This section must be completed when targeting beta to a release.
 Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
+kube-controller-manager needs to be restarted, the CIDR allocator will be switched 
+to the new range allocator based on `--cidr-allocator-type=MultiCIDRRangeAllocator` 
+flag. Rollout can fail if the `--cluster-cidr` field is updated during the 
+kube-controller-manager restart.
+
+Already running workloads will not be impacted.
```

Comment:
L807: What metrics should inform the rollback - please fill in.

Reply:
Done